CONTRIBUTING.md

```diff
@@ -84,6 +84,22 @@ To contribute a patch, the workflow is as follows:
   1. Create topic branch
   1. Commit patches
 
+For GUI related issues or pull requests, the https://github.com/bitcoin-core/gui repository should be used.
+For all other issues and pull requests, the https://github.com/bitcoin/bitcoin main repository should be used.
```

Comment:
for clarity and to avoid centralisation, would suggest avoiding the term "main" in favor of bitcoin-core/gui and bitcoin/bitcoin

Reply:
"main repository" seems like a good way to abbreviate "bitcoin/bitcoin repository" or "non-gui repository".

Switched to "core repository". Hope that's ok as well